src/itemObjGen.txt

```diff
@@ -4035,13 +4078,13 @@
   crafting[]
     ingredients[]
       name = Reinforced Plank
```

Comment:
This should be `Wood Shaft` not Reinforced Plank.

Reply:
Great catch!